runtime.go

```diff
@@ -66,6 +67,7 @@ type runtime struct {
 	stackLimit   int
 	traceLimit   int
 	lck          sync.Mutex
+	stdoutWriter io.Writer
```

Comment:
suggestion: make it clear the use vs the value
```suggestion
	console io.Writer
```